docs/build.md

```diff
@@ -28,4 +30,27 @@ cd atomic-swap
 make build
 ```
 
-This creates `swapd`, `swapcli` and `relayer` binaries in the `bin` folder.
+This creates `swapd` and `swapcli` binaries in the `bin` folder.
+
+Note: if you wish to run a bootnode (see [here](./bootnode.md)), run `make build-all`.
+
+## Docker
+
+1. Ensure docker is installed on your machine.
+
+2. Build the docker image:
+```bash
+./scripts/docker/build-docker-image.sh 
+```
+
+3. For an example of how to run `swapd` with docker:
+```bash
+./scripts/docker/example-docker-run.sh 
```

Comment:
I changed the name of that script in my bootnode PR, as you can now potentially run it unchanged if you preset various `SWAPD_*` environment variables. We should probably still refer to them as examples though in the documentation. The scripts are:
```
./scripts/docker-swapd/run-docker-image.sh
./scripts/docker-bootnode/run-docker-image.sh
```